### DISCUSSION TITLE
justify text on html keyboard response

### DISCUSSION THREAD

**Discussion by Quinlanptq** [jspsych/jsPsych#1027] [Q&A] [answered]
Sorry if this has already been answered but whenever I send a string to html keyboard response it overides my formatting and defaults to center formatting every line of text  - how do I get it to honour a formatted string?

Thanks

Philip.

**Comment by becky-gilbert** [comment, marked as answer]
Hi @Quinlanptq! Just to clarify - you want left-justified text (as opposed to moving the text element itself over to the left side of the page)? If so you should be able to do this with in-line style, using `text-align: left;`. In general, any in-line style should take precedence over the style rules that come from the jsPsych CSS file.

You'll only notice the left-alignment when the text-containing element is wider than (at least some of) the text. For instance, when you have multiple lines of text with different lengths (as in the example below), or when the element's width is set to some larger value.
```javascript
var trial = {
  type: 'html-keyboard-response',
  stimulus: '<div style="text-align:left;">This text should be left-justified.<br>More text.</div>',
  choices: jsPsych.ALL_KEYS
};
```
Does that help? Feel free to clarify if I misunderstood your question, or provide some example code.

**Reply by Quinlanptq** [reply]
Thanks Becky - I have finally grasped the nettle and got to grips with a little more html. Your answer helped immensely. I have also made great strides in passing data to JSPsych via json files. This works a treat and much easier to do than trying to work with requests and promises. Best Philip
